features/Kernel.php

```diff
@@ -76,8 +76,8 @@ protected function configureRoutes(RouteCollectionBuilder $routes): void
             $routes->import('@TwigBundle/Resources/config/routing/errors.xml', '/_error');
         }
 
-        $routes->import('.', null, 'docusign');
-        $routes->import('@TestBundle/Controller', null, 'annotation');
+        $routes->import('.', '/', 'docusign');
+        $routes->import('@TestBundle/Controller', '/', 'annotation');
```

Comment:
not in this PR, but I guess this might be an option.

Reply:
It's just related to e2e tests, it's not related to the final bundle source code